samples/nrf9160/modem_shell/src/ppp/ppp_settings.c

```diff
@@ -33,32 +34,26 @@ static int ppp_settings_uart_dev_conf_get(struct uart_config *uart_conf)
 {
 	int ret;
 
-	if (ppp_uart_dev) {
-		ret = uart_config_get(ppp_uart_dev, uart_conf);
-		if (ret != 0) {
-			mosh_warn("Cannot get PPP uart config, ret %d", ret);
-		}
-	} else {
-		mosh_warn("No PPP uart device with name %s", CONFIG_NET_PPP_UART_NAME);
-		ret = -1;
+	ret = uart_config_get(ppp_uart_dev, uart_conf);
+	if (ret != 0) {
+		mosh_error("Cannot get PPP uart config, ret %d", ret);
```

Comment:
Why `mosh_warn` is changed to `mosh_error` here? There is another call to `mosh_error` done when the above function returns a non-zero value, so I guess this `mosh_warn` here was intentional.
Same story in `ppp_settings_uart_dev_conf_set`.

Reply:
converted back to `mosh_warn`